src/encoded/region_search.py

```diff
@@ -329,12 +328,11 @@ def region_search(context, request):
             peak_query = get_peak_query(start, end, with_inner_hits=True, within_peaks=region_inside_peak_status)
         else:
             peak_query = get_peak_query(start, end, within_peaks=region_inside_peak_status)
-
         peak_results = snp_es.search(body=peak_query,
                                      index=chromosome.lower(),
                                      doc_type=_GENOME_TO_ALIAS[assembly],
                                      size=99999)
-    except Exception:
+    except Exception as e:
```

Comment:
do we not want to return the 'e' in the error?

Reply:
I would think so, but the e is not actually printed, logged or stored.